api/services/authservice.js

```diff
@@ -0,0 +1,70 @@
+const crypto = require('crypto');
+const pool = require('../db/queries');
+const jwt = require('jsonwebtoken');
+const config = require('../config');
+
+class AuthService {
+    constructor() {
+        this.accessTokenSecret = config.accessToken;
+    }
+
+    async SignUp(name, email, password) {
+        const passwordHashed = this.hashPassword(password);
+
+        await pool.query('SELECT * FROM users WHERE email = $1', [email])
```

Comment:
soooo
u are using async here, but await do nothing at all in this case
use async/await appropriatly here